app/views/notifier/order_status_changed_to_approved.html.haml

```diff
@@ -1,16 +1,18 @@
 %h1= @order_detail.facility
 
-%p The following reservation has been approved.
+%p The following order has been approved.
```

Comment:
I18n? Only asking because it's used twice and there's barely anything left to localize in these templates.